maxmind_connector.py

```diff
@@ -249,12 +249,13 @@ def _handle_update_db(self, param):
         self.debug_print('Updating database.')
 
         try:
-            status, msg = self._download_and_replace_db()
+            status, msg, response_headers = self._download_and_replace_db()
             action_result.set_status(status, msg)
         except Exception as e:
             error_msg = 'Error in downloading or replacing database.'
             action_result.set_status(phantom.APP_ERROR, error_msg, e)
 
+        action_result.add_data(dict(response_headers))
```

Comment:
Simply include the headers in the final result. 
This is what the final result looks like:
![image](https://user-images.githubusercontent.com/82124762/147983316-dc32358c-c194-44e1-8323-876778f83663.png)